cli/args/mod.rs

```diff
@@ -573,7 +573,7 @@ fn discover_npmrc(
   // TODO(bartlomieju): update to read both files - one in the project root and one and
   // home dir and then merge them.
   // 3. Try `.npmrc` in the user's home directory
-  if let Some(home_dir) = sys_traits::impls::RealSys.env_home_dir() {
+  if let Some(home_dir) = crate::sys::CliSys::default().env_home_dir() {
```

Comment:
This is fine for now. It won't compile when building deno_rt, but this code isn't used in deno_rt.